test/hotspot/jtreg/vmTestbase/nsk/jdb/locals/locals002/locals002.java

```diff
@@ -88,72 +90,58 @@ public static int run(String argv[], PrintStream out) {
     static final String PACKAGE_NAME = "nsk.jdb.locals.locals002";
     static final String TEST_CLASS = PACKAGE_NAME + ".locals002";
     static final String DEBUGGEE_CLASS = TEST_CLASS + "a";
-    static final String FIRST_BREAK        = DEBUGGEE_CLASS + ".main";
-    static final String LAST_BREAK         = DEBUGGEE_CLASS + ".lastBreak";
+    static final String FIRST_BREAK = DEBUGGEE_CLASS + ".main";
+    static final String LAST_BREAK = DEBUGGEE_CLASS + ".lastBreak";
     static final String COMPOUND_PROMPT_IDENT = "main";
-    static final int    BREAKPOINT_LINE1 = 84;
-    static final int    BREAKPOINT_LINE2 = 100;
-
-    static final String LOCALS[][] = new String[][] {
-       { "boolVar"  , "true"         , "false"         },
-       { "byteVar"  , "27"           , "12"            },
-       { "charVar"  , "V"            , "A"             },
-       { "shortVar" , "767"          , "327"           },
-       { "intVar"   , "1474"         , "3647"          },
-       { "longVar"  , "21345"        , "65789"         },
-       { "floatVar" , "3.141"        , "4.852"         },
-       { "doubleVar", "2.578"        , "3.8976"        },
-       { "objVar"   , "objVarString" , "objArgString"  },
-       { "arrVar"   , "int[5]"       , "int[3]"        }
-
-                                                    };
+    static final int BREAKPOINT_LINE1 = 84;
+    static final int BREAKPOINT_LINE2 = 100;
+
+    static final String[][] LOCALS = new String[][]{
+            {"boolVar", "true", "false"},
+            {"byteVar", "27", "12"},
+            {"charVar", "V", "A"},
+            {"shortVar", "767", "327"},
+            {"intVar", "1474", "3647"},
+            {"longVar", "21345", "65789"},
+            {"floatVar", "3.141", "4.852"},
+            {"doubleVar", "2.578", "3.8976"},
+            {"objVar", "objVarString", "objArgString"},
+            {"arrVar", "int[5]", "int[3]"}
```

Comment:
This table looked a bit better before. Maybe commas should move to the left, but tabbing restored?